src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,67 @@
+package duke.task;
+import java.util.List;
+import java.util.ArrayList;
+/**
+ * Abstract class Task for Todo, Deadline, and Event to inherit from
+ *
+ * @author Pearl Twe
+ * @version CS2103T AY22/23 Semester 2
+ */
```

Comment:
I like that you have the the class description, however I think the description should follow the structure that start with a verb